### DISCUSSION TITLE
This happens when I try to run the game

### DISCUSSION THREAD

**Discussion by KittyDoesThings** [Monika-After-Story/MonikaModDev#9911] [QA - tech support + general]
![Screen_Shot_2023-02-14_at_4 11 25_PM](https://user-images.githubusercontent.com/125409115/218866513-1b4d198e-a12c-4eb8-99f8-403c4d46746e.png)

**Comment by Wingdinggaster656** [comment]
This means you did a bad installation.

Consider using the installer, or strictly follow the instructions.

**Comment by Booplicate** [comment]
To add to the above, also consider switching to a non-steam install.
